Bottom: 0409c41f5aa9698dea2ab9fde8291d0fad626fd7
Top:    68924b100a93e4f53bc3e5c917b20d7dc61c58af
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:31:08 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/lib/master.c b/lib/master.c
index 4341003..491fda0 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1394,11 +1394,10 @@ int master_list_empty(struct master *master)
 	return res;
 }
 
-int master_done(struct master *master)
+void master_done(struct master *master)
 {
 	struct list_head *head, *p;
 	struct master_mapent *entry;
-	int res = 0;
 	int status;
 
 	finish_mutex_lock();
@@ -1421,12 +1420,7 @@ int master_done(struct master *master)
 	if (status)
 		fatal(status);
 
-	if (sdc.busy)
-		res = 1;
-
 	finish_mutex_unlock();
-
-	return res;
 }
 
 inline unsigned int master_get_logopt(void)
